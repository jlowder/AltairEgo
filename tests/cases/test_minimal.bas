10 REM MINIMAL TEST FOR DEBUGGING
20 PRINT "Testing basic operations..."
30 A=5: B=3
40 PRINT "A="; A; " B="; B
50 PRINT "A+B="; A+B
60 DIM X(5)
70 X(0)=10: X(1)=20
80 PRINT "X(0)="; X(0); " X(1)="; X(1)
90 FOR I=1 TO 3: PRINT I;: NEXT I: PRINT
100 IF A>B THEN PRINT "A is greater"
110 GOSUB 200
120 END
200 PRINT "In subroutine"
210 RETURN
