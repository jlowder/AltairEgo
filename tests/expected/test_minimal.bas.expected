Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
Testing basic operations...
A= 5  B= 3
A+B= 8
X(0)= 10  X(1)= 20
 1  2  3
A is greater
In subroutine
OK
